testtracer_test.go

```diff
@@ -0,0 +1,48 @@
+package opentracing
+
+// testTracer is a most-noop Tracer implementation that makes it possible for
+// unittests to verify whether certain methods were / were not called.
+type testTracer struct{}
```

Comment:
Why do you need all of this? Just use `basictracer` with an `InMemoryRecorder`, and finish the child span which gives you access to `RawSpan.Context.ParentID`.